### PR TITLE
add Integer Consulting and Inoweiser

### PR DESCRIPTION
**Pull request by gustavo-fernandes**

### CODE REVIEW

**Comment by vpereira01** [comment]
@gustavo-fernandes cleaned up the PR commits since it contained unnecessary extra stuff.

**Review by vpereira01 — DISMISSED** [review]
:+1:

File: README.md [hunk 1]
```diff
@@ -114,6 +114,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [Dellent Consulting](https://dellentconsulting.com/) [:rocket:](https://dellentconsulting.com/_carreiras/) | Consultancy, Outsourcing. | `Aveiro` `Lisboa` |
 | [Everis](https://www.everis.com/portugal/pt-pt/home-pt) | Consultancy, Outsourcing. | `Lisboa` |
 | [GLanDrive](https://www.glandrive.pt) | Consultancy, Outsourcing. | `Lisboa` `Porto` |
+| [Integer Consulting](https://integerconsulting.pt/) [:rocket:](https://integerconsulting.pt/opportunity/) | Consultancy, Outsourcing and Nearshore | `Lisboa` `Porto` <br> `São Paulo` |
```
Comment (marmelo) [inline README.md:117]: Only PT cities are relevant.

```suggestion
| [Integer Consulting](https://integerconsulting.pt/) [:rocket:](https://integerconsulting.pt/opportunity/) | Consultancy, Outsourcing and Nearshore | `Lisboa` `Porto` |
```

**Review by marmelo — CHANGES_REQUESTED** [review]

**Comment by gustavo-fernandes** [comment]
done

File: README.md [hunk 2]
```diff
@@ -177,7 +178,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [Reviewpad](https://www.reviewpad.com) [:rocket:](https://reviewpad.com/discord/hiring) [:octocat:](https://github.com/reviewpad) | Pull request automations to safely merge faster with happy developers. | `Porto` <br> `Remote` |
 | [Rows](https://www.rows.com) [:rocket:](https://github.com/dashdash/hiring) | Create interactive web apps using spreadsheet skills. | `Porto` |
 | [Synopsys](https://www.synopsys.com) [:rocket:](https://synopsys.ongig.com/?l=Portugal&regions=EMEA) | Silicon chip design, verification, IP integration and application security | `Lisbon` `Porto` |
-| [Unbabel](https://unbabel.com) [:rocket:](https://unbabel.com/careers/) | Translation as a Service. | `Lisboa` |
+| [Unbabel](https://unbabel.com) [:rocket:](https://unbabel.com/careers/) [:octocat:](https://github.com/Unbabel) | Translation as a Service. | `Lisboa` |
```
Comment (marmelo) [inline README.md:181]: I guess there are changes here from other PRs.
Can you please check if you are rebased with the current master branch?

**Review by marmelo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (gustavo-fernandes) [reply README.md]: added Inoweiser too

Comment (vpereira01) [reply README.md]: Rebase was not done, looked like a merge from upstream which added more other commits.
I did rebase and fixed commits wording.

**Review by vpereira01 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (gustavo-fernandes) [reply README.md]: @vpereira01 Thanks

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**